client/checkout/constants.js

```diff
@@ -1,7 +1,11 @@
 export const PAYMENT_METHOD_NAME_CARD = 'woocommerce_payments';
-export const PAYMENT_METHOD_NAME_BECS = 'woocommerce_payments_becs';
+export const PAYMENT_METHOD_NAME_BANCONTACT = 'woocommerce_payments_bancontact';
+export const PAYMENT_METHOD_NAME_BECS = 'woocommerce_payments_au_becs_debit';
+export const PAYMENT_METHOD_NAME_EPS = 'woocommerce_payments_eps';
 export const PAYMENT_METHOD_NAME_GIROPAY = 'woocommerce_payments_giropay';
-export const PAYMENT_METHOD_NAME_SEPA = 'woocommerce_payments_sepa';
+export const PAYMENT_METHOD_NAME_IDEAL = 'woocommerce_payments_ideal';
+export const PAYMENT_METHOD_NAME_P24 = 'woocommerce_payments_p24';
+export const PAYMENT_METHOD_NAME_SEPA = 'woocommerce_payments_sepa_debit';
```

Comment:
I just saw these in the code: 
https://github.com/Automattic/woocommerce-payments/blob/23a3b08ba18f5dc855798d233267dd44baf1ac3b/includes/payment-methods/class-upe-payment-gateway.php#L979-L987

It looks like the existing code base preferred "class constants" over "a constant file". Maybe we should take these out and use the class constant approach?